src/backend/mod.rs

```diff
@@ -112,6 +112,10 @@ pub trait Backend {
     /// Get the size of the terminal screen as a [`Rect`].
     fn size(&self) -> Result<Rect, io::Error>;
 
+    /// Get the size of the terminal screen as well as windows sixe in pixels as a [`Rect`] and a
+    /// `(u16, u16)` tuple.
+    fn window_size(&mut self) -> Result<(Rect, (u16, u16)), io::Error>;
```

Comment:
Do you think it makes sense to split this to two methods with more explicit names instead of a single returning a tuple? Is the Rect ever not positioned at x=0, y=0?

Reply:
I say no, because 2 out of 3 implementations return both things at once in one call, due to the underlying API.

Reply:
Oh and the `Rect` is just like `size`, always x=0,y=0.